bridges/TikTokBridge.php

```diff
@@ -0,0 +1,87 @@
+<?php
+class TikTokBridge extends BridgeAbstract {
+	const NAME = 'TikTok Bridge';
+	const URI = 'https://www.tiktok.com';
+	const DESCRIPTION = 'Returns posts';
+	const MAINTAINER = 'VerifiedJoseph';
+	const PARAMETERS = array(
+		'By user' => array(
+		'username' => array(
+			'name' => 'Username',
+			'type' => 'text',
+			'required' => true,
+			'exampleValue' => '@tiktok',
+		)
+	));
+
+	const TEST_DETECT_PARAMETERS = array(
+		'https://www.tiktok.com/@tiktok' => array(
+			'context' => 'By user', 'username' => '@tiktok'
+		)
+	);
+
+	const CACHE_TIMEOUT = 900; // 15 minutes
+
+	private $feedName = '';
+
+	public function detectParameters($url) {
+		$params = array();
```

Comment:
Move this into the if. No need to initialize the array if there is no match. In fact, maybe initialize it with its contents rather than setting them separately.